addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/SpiderJob.java

```diff
@@ -42,14 +43,11 @@
 
     private static final String PARAM_CONTEXT = "context";
     private static final String PARAM_URL = "url";
-    private static final String PARAM_FAIL_IF_LESS_URLS = "failIfFoundUrlsLessThan";
-    private static final String PARAM_WARN_IF_LESS_URLS = "warnIfFoundUrlsLessThan";
     private static final String PARAM_MAX_DURATION = "maxDuration";
 
     private ExtensionSpider extSpider;
 
-    private int failIfFoundUrlsLessThan = 0;
-    private int warnIfFoundUrlsLessThan = 0;
+    private AutomationStatisticTest urlsFoundTest;
```

Comment:
I'm confused - why do we need this? I was expecting this to just be a generic test so no specific code in the job :/